### PR TITLE
Bring NetApp Files to R3.5 standard

### PR DESCRIPTION
**Pull request by leonardbf**
Includes export policy and active directory.
Still in preview - breaking changes are not an issue

Swagger:
https://github.com/leonardbf/azure-rest-api-specs/tree/master/specification/netapp/resource-manager

Last swagger PR:
Azure/azure-rest-api-specs#5446

### CODE REVIEW

**Comment by leonardbf** [comment]
@dsgouda you have added the changes requested label but I don't see any actions stated. Is there something I'm required to do? Thanks.

**Comment by dsgouda** [comment]
@lebronJ Please pull down latest changes from master and resolve the conflicts or recreate the changes branch and open a new PR

**Comment by leonardbf** [comment]
@dsgouda I have pulled from upstream into my branch.

**Comment by dsgouda** [comment]
@leonardbf I missed that you were targeting the psSdkJson6 branch, we have switched our default branch from psSdkJson6 to master. Please update the branch again with master, I shall post my review in the meantime.

File: src/SDKs/NetApp/Management.NetApp/Microsoft.Azure.Management.NetApp.csproj [hunk 1]
```diff
@@ -26,7 +26,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
```
Comment (dsgouda) [inline src/SDKs/NetApp/Management.NetApp/Microsoft.Azure.Management.NetApp.csproj]: Was this intentional?

File: src/SDKs/NetApp/Management.NetApp/Microsoft.Azure.Management.NetApp.csproj [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <Description>Provides NetApp storage management capabilities for Microsoft Azure.</Description>
     <AssemblyTitle>Microsoft Azure NetApp Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.NetApp</AssemblyName>
-    <Version>0.9.0-preview</Version>
+    <Version>0.9.1-preview</Version>
     <PackageTags>MicrosoftAzure Management;NetApp</PackageTags>
     <PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/NetApp/Management.NetApp/Microsoft.Azure.Management.NetApp.csproj]: Please update the `PackageReleaseNotes`

File: src/SDKs/NetApp/NetApp.Tests/NetApp.Tests.csproj [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <PackageId>NetApp.Tests</PackageId>
     <Description>NetApp.Tests Class Library</Description>
     <AssemblyName>NetApp.Tests</AssemblyName>
-    <Version>1.0.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/NetApp/NetApp.Tests/NetApp.Tests.csproj]: Test projects are never published. Please set this to `1.0.0` and do not modify henceforth

**Review by dsgouda — CHANGES_REQUESTED** [review]
Left a few comments

File: src/SDKs/NetApp/Management.NetApp/Microsoft.Azure.Management.NetApp.csproj (hunk 1 above)
Comment (leonardbf) [reply src/SDKs/NetApp/Management.NetApp/Microsoft.Azure.Management.NetApp.csproj]: No. I shall change it back.

File: src/SDKs/NetApp/NetApp.Tests/NetApp.Tests.csproj (hunk 3 above)
Comment (leonardbf) [reply src/SDKs/NetApp/NetApp.Tests/NetApp.Tests.csproj]: ok. Done.

**Comment by leonardbf** [comment]
@dsgouda Yes I was unaware the target should now be master. I have pulled latest from upstream master, merged my changes there and created a new PR https://github.com/Azure/azure-sdk-for-net/pull/5646. It includes changes for your comments above. I will close this PR when you have responded concerning the new one.

**Pull request closed without merging**